model.py

```diff
@@ -6592,17 +6592,13 @@ def fetch(cls, _db, lane, type, facets, pagination, annotator,
                 max_age = 0
         if isinstance(max_age, int):
             max_age = datetime.timedelta(seconds=max_age)
+
+        unique_key = None
         if lane and isinstance(lane, Lane):
             lane_id = lane.id
-            if entrypoint:
-                unique_key = entrypoint.INTERNAL_NAME
-            else:
-                unique_key = None
```

Comment:
Since entry point is now a facet group, the information necessary to distinguish between the same feed in two entry points is now kept in the facet field and there's no need for this code.